R/react.R

```diff
@@ -118,16 +137,21 @@ ReactiveEnvironment <- R6Class(
     hasPendingFlush = function() {
       return(!.pendingFlush$isEmpty())
     },
+    # Returns TRUE if anything was actually called
```

Comment:
I'm curious: do we actually use the return values anywhere?

Reply:
I think I was calling it in a while loop originally??